src/demo/demoUtils.ts

```diff
@@ -31,7 +31,7 @@ export type QueryParams = {
   showUserConsent?: StringifiedBoolean
   smsNumber?: StringifiedBoolean
   snapshotInterva?: StringifiedBoolean
-  snapshotInterval?: StringifiedBoolean
+  snapshotInterval?: string
```

Comment:
These feel like they might be a duplicated type, but just with/without typo? 🤔

Reply:
Yeah it seems that `snapshotInterva` is both redundant and a typo!